socs/agents/scpi_psu/agent.py

```diff
@@ -104,6 +136,31 @@ def stop_monitoring(self, session, params=None):
         self.monitor = False
         return True, "Stopping current monitor"
 
+    @ocs_agent.param('channel', type=int, choices=[1, 2, 3])
+    def get_voltage(self, session, params=None):
+        """get_voltage(channel)
+
+        **Task** - Measure and return the voltage of the power supply.
+
+        Parameters:
+            channel (int): Channel number (1, 2, or 3).
+
+        """
+        chan = params['channel']
+        with self.lock.acquire_timeout(1) as acquired:
+            if acquired:
+                data = {
+                    'timestamp': time.time(),
+                    'block_name': 'output',
+                    'data': {}
+                }
+
+                data['data']['Voltage_{}'.format(chan)] = self.psu.get_volt(chan)
+                session.data = data
```

Comment:
The structure here is the one required for publishing to an [ocs Feed](https://ocs.readthedocs.io/en/main/developer/feeds.html#recorded-feed-message-format). This isn't really required for the `session.data`, which is very flexible. Since this is just an instance of retrieving a single number I'd suggest simplifying this down to:
```
data = {
    'timestamp': time.time(),
    'voltage': self.psu.get_volt(chan),
}

session.data = data
```

Whatever the format of the `session.data` dict, I'd ask that you put an example in the docstring for the method. (A good reference on documentation in OCS here: https://ocs.readthedocs.io/en/main/developer/agent_references/documentation.html#task-and-process-documentation -- and again sorry that this old agent isn't up to these newer standards, so the local context was a bit lacking...)

Reply:
In order to retain the channel information in the output, could I do something like
```
data = {
    'timestamp': time.time()
}
data['Voltage_{}'.format(chan)] = self.psu.get_volt(chan)
session.data = data
```

Reply:
I think I'd rather see it as a separate key/value pair, i.e.:
```
data = {
    'timestamp': time.time(),
    'channel' = chan,
    'voltage' = self.psu.get_volt(chan),
}

session.data = data
```

But yeah, still fine to include if you'd like.

Reply:
Ok, changed this for both the `get_voltage()` and `get_current()` and I updated the docstrings to reflect the new output.